core/common/src/thrift/common.thrift

```diff
@@ -28,6 +28,11 @@ enum CommandType {
   Persist = 5,  // Ask the worker to persist a file for lineage
 }
 
+enum TTtlAction {
```

Comment:
this could just be an i32, recording the mapping from wire class enums to ints in the toThrift() and fromThrift() methods